app/models/video.rb

```diff
@@ -7,4 +8,10 @@ def self.search_by_title(search_term)
     return [] if search_term.blank?
     Video.where('lower(title) LIKE ?', "%#{search_term}%").order('created_at DESC')
   end
+
+  def calculate_rating_average
```

Comment:
Looks good but ActiveRecord has built in method `.average` - read more [here](http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-average)